test/unit/pools/fixedRate/FixedRateExchange.test.ts

```diff
@@ -515,67 +504,68 @@ describe('Fixed Rate unit test', () => {
         ).baseTokenAmount
       ).to.equal('100.3')
     })
+
     it('#getAmountBTOut - should get bt amount out for a specific dt amount', async () => {
       // 99.8 USDC for 100 DT (0.1% market fee and 0.1% ocean fee)
       expect(await fixedRate.getAmountBTOut(exchangeId, '100')).to.equal('99.7')
     })
 
-    it('#buyDT - user2 should buy some dt', async () => {
+    it('#buyDT - user1 should buy some dt', async () => {
       // total supply is ZERO right now so dt owner mints 1000 DT and approves the fixed rate contract
       await dtContract.methods
         .mint(exchangeOwner, web3.utils.toWei('1000'))
         .send({ from: exchangeOwner })
       await dtContract.methods
-        .approve(fixedRateAddress, web3.utils.toWei('1000'))
+        .approve(contracts.fixedRateAddress, web3.utils.toWei('1000'))
         .send({ from: exchangeOwner })
-      // user2 gets 100 USDC so he can buy DTs
-      await usdcContract.methods.transfer(user2, 100 * 1e6).send({ from: exchangeOwner })
+      // user1 gets 100 USDC so he can buy DTs
+      await usdcContract.methods.transfer(user1, 100 * 1e6).send({ from: exchangeOwner })
```

Comment:
shouldn't we use here the decimals helpers from TokenUtils to convert the values ?